src/main/java/teammates/logic/core/FeedbackSessionsLogic.java

```diff
@@ -50,7 +50,34 @@
 import teammates.storage.api.StudentsDb;
 
 public class FeedbackSessionsLogic {
-
+    
+    private static final String ASSUMPTION_FAIL_DELETE_INSTRUCTOR = "Fail to delete instructor respondant for ";
+    private static final String ASSUMPTION_FAIL_RESPONSE_ORIGIN = "Client did not indicate the origin of the response(s)";
+    private static final String ERROR_NUMBER_OF_RESPONSES_EXCEEDS_RANGE = "Number of responses exceeds the limited range";
+    private static final String ERROR_SENDING_EMAILS = "Error while sending emails :";
+    private static final String ERROR_SESSION_UNPUBLISH_ALREADY = "Session is already unpublished.";
+    private static final String ERROR_SESSION_UNPUBLISH_PRIVATE = "Private session can't be unpublished.";
+    private static final String ERROR_SESSION_PUBLISH_ALREADY = "Session is already published.";
+    private static final String ERROR_SESSION_PUBLISH_PRIVATE = "Private session can't be published.";
+    private static final String KEY_FROM_SECTION = "fromSection";
+    private static final String KEY_IN_SECTION = "inSection";
+    private static final String KEY_IS_INCLUDE_RESPONSE_STATUS = "isIncludeResponseStatus";
+    private static final String KEY_QUESTION_ID = "questionId";
+    private static final String KEY_RANGE = "range";
+    private static final String KEY_SECTION = "section";
+    private static final String KEY_TO_SECTION = "toSection";
+    private static final String KEY_VIEW_TYPE = "viewType";
```

Comment:
I think `KEY_....` should actually be `PARAM_...`

Also, we can group the `PARAM_...` constants either above or below the message-related constants, separated by an empty line.